.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Context
+
+(Is your feature request related to a problem?)  
+*Ex: I get frustrated when having to use nested `if` conditions where a `elif` could be used*
+
+# Description of the feature
```

Comment:
How is this different from context?

Reply:
I merged the two of them in https://github.com/inmanta/inmanta/pull/2416/commits/2e2f1f6a155d159d30daa2953c86835f4bde579f